src/Listbox.js

```diff
@@ -324,6 +340,17 @@ export const Listbox = {
       this.$data.listboxButtonRef.value().focus()
     },
     select(value) {
+      if (this.$props.multiple) {
+        const values = this.$props.value
+        const index = values.indexOf(value)
```

Comment:
I'd love to clean this up a bit to be more human-readable.

My suggestion:

```js

if (this.$props.multiple) {
	const values = this.$props.value
	// If a value is already selected we need to deselect it,
	// else we need to add it to the list of selected values
	const newValues = values.includes(value) ? values.filter(v => v !== value) : [...values, value]
	this.$emit('input', newValues)
	return
}
```

Reply:
That is actually even more performant 😄 Thanks!